app/models/teacher_training_adviser/steps/retake_gcse_maths_english.rb

```diff
@@ -2,7 +2,7 @@ module TeacherTrainingAdviser::Steps
   class RetakeGcseMathsEnglish < Wizard::Step
     attribute :planning_to_retake_gcse_maths_and_english_id, :integer
 
-    validates :planning_to_retake_gcse_maths_and_english_id, types: { method: :get_candidate_retake_gcse_status, message: "You must select either yes or no" }
+    validates :planning_to_retake_gcse_maths_and_english_id, types: { method: :get_candidate_retake_gcse_status, message: "Select yes if you are planning to retake either English or maths (or both) GCSEs, or equivalent" }
```

Comment:
Is the casing not a bit off in some of theses? I would have expected this to be:

```...either English or Maths (or both) GCSEs, or equivalent```

or

```...either english or maths (or both) GCSEs, or equivalent```

Reply:
copied it from the question, not sure, Darren approved them though

Reply:
could be wrong in the question also